reflex/components/datadisplay/code.py

```diff
@@ -411,6 +411,12 @@ class CodeBlock(Component):
     # Props passed down to the code tag.
     code_tag_props: Var[Dict[str, str]]
 
+    # Whether a copy button should appear.
```

Comment:
In this approach, do these extra props end up getting compiled in? If so, I feel we should have a way to distinguish between these reflex-defined props vs the ones that get compiled in at the end.

Reply:
I think for these cases we can add them to the excluded props so they don't get compiled:

```python
class Code(rx.Component):
  ...
  
  def _exclude_prop(self) -> list[str]:
      return ["code_tag_props"]

```